Second

Solpostgit opened the pull request:
Проверьте ДЗ

Solpostgit committed:
Добавил класс Hogwarts.

Solpostgit committed:
Добавил поля и методы в класс Hogwarts.

Solpostgit committed:
Добавил класс Griffindor.
Внёс поля и методы.

Solpostgit committed:
Добавил класс Hufflepuff.
Внёс поля и методы.

Solpostgit committed:
Добавил класс Ravenclaw.
Внёс поля и методы.

Solpostgit committed:
Создал массив объектов printedStudents.

Solpostgit committed:
Создал массив объектов printedStudents.

Solpostgit committed:
Метод toString в классе Slytherin

Solpostgit committed:
Создал класс методов PrintService.

Solpostgit committed:
Написал метод вывода данных студента printStudent()

Solpostgit committed:
Написал метод сравнения Гриффиндорцев compareFaculty().

Solpostgit committed:
Использую перегрузку метода compareFaculty для Hufflepuff.

Solpostgit committed:
Использую перегрузку метода compareFaculty для Ravenclaw.

Solpostgit committed:
Использую перегрузку метода compareFaculty для Slytherin.

Solpostgit committed:
Написал метод сравнения двух студентов по силе магии и трансгрессии.

Solpostgit committed:
Готово

Solpostgit committed:
Написал комментарии к коду

OlgaLugacheva commented on src/Hogwarts.java:
```diff
@@ -0,0 +1,56 @@
+public class Hogwarts {
+    String name;
+    String faculty;
+    int powerMagic;
+    int transgresDist;
```
модификаторы доступа все равно нужны)

OlgaLugacheva commented on src/Main.java:
```diff
@@ -1,5 +1,101 @@
\ No newline at end of file
+
+    //Метод сравнения Гриффендорцев
+    public static void compareFaculty(Griffindor object1, Griffindor object2) {
```
object1 и object2 не слишком понятные в контексте названия параметров, лучше  griffindorStudent1

Solpostgit committed:
Добавил модификатор доступа protected для полей класса Hogwarts.
Исправил в методах compareFaculty, compareStudent заменил названия объектов на понятные.

Solpostgit commented:
Внёс исправления.
